src/app/users/profile/profile.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
This profile is doing too many things atm.  Drop the ask a question functionality, display topics in a similar way to other places on the site (with chips!)... I think Interests are just supposed to be tags, not any interest they want to add.  So something like the self-tags component functionality, but for associates.

This should definitely be in a different PR, because actually finishing the functionality/display will take awhile